Mathlib/Algebra/Polynomial/Bivariate.lean

```diff
@@ -41,15 +41,101 @@ abbrev CC (r : R) : R[X][Y] := C (C r)
 lemma evalEval_C (x y : R) (p : R[X]) : (C p).evalEval x y = p.eval x := by
   rw [evalEval, eval_C]
 
+@[simp]
+lemma evalEval_CC (x y : R) (p : R) : (CC p).evalEval x y = p := by
+  rw [evalEval_C, eval_C]
+
+@[simp]
+lemma evalEval_zero (x y : R) : (0 : R[X][Y]).evalEval x y = 0 := by
+  simp only [evalEval, eval_zero]
+
+@[simp]
+lemma evalEval_one (x y : R) : (1 : R[X][Y]).evalEval x y = 1 := by
+  simp only [evalEval, eval_one]
+
+@[simp]
+lemma evalEval_ofNat (x y : R) (n : ℕ) [n.AtLeastTwo] :
+    (OfNat.ofNat n : R[X][Y]).evalEval x y = OfNat.ofNat n := by
+  simp only [evalEval, eval_ofNat]
+
+@[simp]
+lemma evalEval_natCast (x y : R) (n : ℕ) : (n : R[X][Y]).evalEval x y = n := by
+  simp only [evalEval, eval_natCast]
+
+@[simp]
 lemma evalEval_X (x y : R) : X.evalEval x y = y := by
   rw [evalEval, eval_X, eval_C]
 
+@[simp]
+lemma evalEval_add (x y : R) (p q : R[X][Y]) :
```

Comment:
Ugh, this is not a simp lemma for MvPolynomial, but is for Polynomial. Given that inconsistency, I don't mind what happens here, but if you feel like looking into that later, that would be wonderful. :-)